crd/multitenancy/api/v1alpha1/podnetworkinstance.go

```diff
@@ -36,19 +35,35 @@ type PodNetworkInstanceList struct {
 	Items           []PodNetworkInstance `json:"items"`
 }
 
+// PodNetworkInstanceItem describes a template for how to attach a PodNetwork to a Pod
+type PodNetworkInstanceItem struct {
+	// PodNetwork is the name of a PodNetwork resource
+	PodNetwork string `json:"podNetwork"`
+	// PodIPReservationSize is the number of IP address to statically reserve
+	// +kubebuilder:default=0
+	PodIPReservationSize int `json:"podIPReservationSize,omitempty"`
+}
+
 // PodNetworkInstanceSpec defines the desired state of PodNetworkInstance
 type PodNetworkInstanceSpec struct {
-	// pod network resource object name
-	PodNetwork string `json:"podnetwork"`
-	// number of backend IP address to reserve for running pods
+	// Deprecated - use PodNetworks
+	// +kubebuilder:validation:Optional
+	PodNetwork string `json:"podnetwork,omitempty"`
+	// Deprecated - use PodNetworks
 	// +kubebuilder:default=0
-	PodIPReservationSize int `json:"podIPReservationSize"`
+	PodIPReservationSize int `json:"podIPReservationSize,omitempty"`
+	// PodNetworks describes each PodNetwork to attach to a single Pod
+	// optional for now in case orchestrator uses the deprecated fields
+	// +kubebuilder:validation:Optional
+	PodNetworks []PodNetworkInstanceItem `json:"podNetworks"`
```

Comment:
Suggestion: rename this to PodNetworkInstances[]. I think given that we are pushing down legacy fields from PNISpec to PNIItem, it would make sense to keep the same name for each of the items since the still represents the same idea of a 'reservation' instead of PodNetwork that represents a subnet registration

Reply:
I think it may be more confusing for a PodNetworkInstance to contain PodNetworkInstances

Reply:
I agree this is confusing as-is, going to rename to `PodNetworkConfigs`